### PR TITLE
update readme; add contributing page

### PR DESCRIPTION
**Pull request by nreinicke**
Updates the docs a bit:
 - add more context around conda to readme
 - add contributing page for how to fork and open a PR
 - move scenario config section in readme to inputs section in docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -305,7 +287,8 @@ If you have found HIVE useful for your research, please cite our [technical repo
 
 ## Contributors
 
-HIVE is currently maintained by Nick Reinicke ([@nreinicke] (https://github.com/nreinicke)) and Rob Fitzgerald ([@robfitzgerald](https://github.com/robfitzgerald)). It would not be what it is today without the support of: 
+HIVE is currently maintained by Nick Reinicke ([@nreinicke] (<https://github.com/nreinicke>)) and Rob Fitzgerald ([@robfitzgerald](https://github.com/robfitzgerald)). It would not be what it is today without the support of:
```
Comment (robfitzgerald) [inline README.md:290]: there's a space in the hyperlink for your github account so as a result it doesn't parse correctly:

Nick Reinicke ([@nreinicke] (https://github.com/nreinicke)...

should be: Nick Reinicke ([@nreinicke](https://github.com/nreinicke)...

File: README.md [hunk 2]
```diff
@@ -317,6 +300,6 @@ HIVE is currently maintained by Nick Reinicke ([@nreinicke] (https://github.com/
 
 ## Notice
 
-Copyright © 2022 Alliance for Sustainable Energy, LLC, Inc. All Rights Reserved
+Copyright © 2023 Alliance for Sustainable Energy, LLC, Inc. All Rights Reserved
```
Comment (robfitzgerald) [inline README.md:303]: i _think_ this copyright year shouldn't be carried forward, it should stay fixed at 2022 to coincide with the [software record](https://www.osti.gov/biblio/1896384)

File: docs/source/developer/contributing.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+1. Go to the "Issues" tab in the repository.
+1. Look for issues with labels such as "good first issue" if you're new to the project.
+1. Read the issue description and any associated comments to make sure you understand the problem and its requirements.
+1. Choose an issue that aligns with your skills and interests.
+
+## Letting Others Know You're Working on the Issue
+
+1. Leave a comment on the issue, stating that you'd like to work on it. This helps prevent duplication of effort and lets us know someone is addressing the problem.
+1. Wait for a response from the maintainers or the issue creator. They may provide additional information or guidance, or assign the issue to you.
```
Comment (robfitzgerald) [inline docs/source/developer/contributing.md:22]: we can't assign the issues to people who fork it, can we?

**Review by robfitzgerald — CHANGES_REQUESTED** [review]
this should you manage the sprint. thanks for taking care of it. i have a few changes in here and one question.

File: docs/source/developer/contributing.md (hunk 3 above)
Comment (nreinicke) [reply docs/source/developer/contributing.md:22]: Looks like if they comment on the issue we can assign it to them: https://github.blog/changelog/2019-06-25-assign-issues-to-issue-commenters/

File: README.md (hunk 2 above)
Comment (nreinicke) [reply README.md:303]: That makes sense, I'll revert it

**Pull request merged**